test/test-integration.js

```diff
@@ -86,11 +94,39 @@ marionette('gaia-switch', function() {
   test('gaia-switch after user action', function() {
```

Comment:
I know this isn't part of this patch, but as per the comment below, I'd like to see the test name describe the expected functionality. For example:

```js
test('it becomes checked when the tapped or clicked', ...)
```